dataset/src/test/resources/log4j.properties

```diff
@@ -144,3 +144,6 @@ log4j.logger.org.spark-project.jetty.util.thread.QueuedThreadPool=ERROR
 log4j.logger.org.spark-project.jetty.util.thread.Timeout=ERROR
 log4j.logger.org.spark-project.jetty=ERROR
 log4j.logger.Remoting=ERROR
+
+# To debug expressions:
+#log4j.logger.org.apache.spark.sql.catalyst.expressions.codegen.CodeGenerator=DEBUG
```

Comment:
Could you add an extra line there?

Reply:
There is a comment just on the previous line, I don't see what to add

Reply:
@cchantep oh, I mean an extra whitespace after the line `149`, so the end of the line is not here 
![image](https://user-images.githubusercontent.com/4929546/132126169-bc812946-1e65-4717-a251-7d99886b553a.png)

Reply:
I don't think we should care about that, but ok.